### ISSUE TITLE
Config to make Windows App portable? Don't save under AppData/Roaming

### ISSUE THREAD

**Issue by ChrisPea** [zulip/zulip-desktop#923]
---

- **Operating System**:
  - [x] Windows
  - [ ] Linux/Ubuntu
  - [ ] macOS
- **Clear steps to reproduce the issue**:
Is it possible to change the path to profile? i want to build a preconfigured client without the effort to make an install script. Is there any possibility without changing the sourcecode? 
- **Relevant error messages and/or screenshots**:

**Comment by akashnimare** [comment]
Do you want to change the installation path for the app programmatically, right?

**Comment by ChrisPea** [comment]
> 
> 
> Do you want to change the installation path for the app programmatically, right?

no, i just want to configure the profil path in a propertiefile near the executeable to use the non-installation-edition like a portable edition.

now i am using a workaround with a symbolic link from user/appdata/roaming/zulip to a folder i can give to others, so i can provide a preconfigured zulip profil to my colleagues

**Comment by abhigyank** [comment]
@ChrisPea I understand you're trying to deploy to a lot of user. While the profile path right now is not configurable, you can check out the [Enterprise Deployment](https://github.com/zulip/zulip-desktop/blob/master/docs/Enterprise.md) configuration, if it addresses your issues.
